web/config.py

```diff
@@ -30,15 +30,17 @@
                     each month. This month we've hit our limit. You can still draw art here, but the website won't
                     accept submissions. Come back next month and we'll start fresh!"""
 
-
-MAIL_SERVER = os.environ['EMAIL_SERVER']
-MAIL_PORT = os.environ['EMAIL_PORT']
-MAIL_USE_TLS = False
-MAIL_USE_SSL = True
-# MAIL_DEBUG : default app.debug
-MAIL_USERNAME = os.environ['EMAIL_USER']
-MAIL_PASSWORD = os.environ['EMAIL_PASSWORD']
-MAIL_DEFAULT_SENDER = os.environ['EMAIL_SENDER']
-# MAIL_MAX_EMAILS : default None
-# MAIL_SUPPRESS_SEND : default app.testing
-# MAIL_ASCII_ATTACHMENTS : default False
\ No newline at end of file
+try:
+    MAIL_SERVER = os.environ['EMAIL_SERVER']
+    MAIL_PORT = os.environ['EMAIL_PORT']
+    MAIL_USE_TLS = False
+    MAIL_USE_SSL = True
+    # MAIL_DEBUG : default app.debug
+    MAIL_USERNAME = os.environ['EMAIL_USER']
+    MAIL_PASSWORD = os.environ['EMAIL_PASSWORD']
+    MAIL_DEFAULT_SENDER = os.environ['EMAIL_SENDER']
+    # MAIL_MAX_EMAILS : default None
+    # MAIL_SUPPRESS_SEND : default app.testing
+    # MAIL_ASCII_ATTACHMENTS : default False
+except:
```

Comment:
This does not prevent us from starting the app without the proper env variables. The person starting the app might still miss it. Maybe we could raise after printing that message? This would, however, make it hard to run locally. I could also change this in the refactor instead and for now, deploy as-is.

Reply:
My thinking here is that it _shouldn't_ be a big deal if we haven't set up the email env vars, since it's important for our workflow, but ultimately not a big deal to the core functionality of the app. And for local tests of UI changes, you probably don't actually want emails going out.
**Proposal**: If Email env vars aren't available, we ask for confirmation that we don't want to be able to send emails